client/app/scripts/index.js

```diff
@@ -0,0 +1,237 @@
+import 'angular-vs-repeat';
+import 'ng-file-upload';
+import 'raven-js';
+import 'rangy';
+import 'rangy-saverestore';
+import 'ment.io';
+import 'angular-embed/dist/angular-embed';
+import 'angular-contenteditable';
+import 'angular-messages';
+import 'lr-infinite-scroll';
+
+import _ from 'lodash';
+import moment from 'moment';
+
+_.sortByOrder = require('lodash.sortbyorder');
+
+// This is an ugly little hack required by the venerable superdesk.editor to work
+import MediumEditor from 'medium-editor';
+window.MediumEditor = MediumEditor;
+
+import 'superdesk-core/scripts/core/activity';
+import 'superdesk-core/scripts/core/analytics';
+import 'superdesk-core/scripts/core/api';
+import 'superdesk-core/scripts/core/auth';
+import 'superdesk-core/scripts/core/beta';
+import 'superdesk-core/scripts/core/datetime';
+import 'superdesk-core/scripts/core/error';
+import 'superdesk-core/scripts/core/elastic';
+import 'superdesk-core/scripts/core/filters';
+import 'superdesk-core/scripts/core/services';
+import 'superdesk-core/scripts/core/directives';
+import 'superdesk-core/scripts/core/editor2';
+import 'superdesk-core/scripts/core/spellcheck';
+import 'superdesk-core/scripts/core/editor3';
+import 'superdesk-core/scripts/core/features';
+import 'superdesk-core/scripts/core/list';
+import 'superdesk-core/scripts/core/keyboard';
+import 'superdesk-core/scripts/core/privileges';
+import 'superdesk-core/scripts/core/notification';
+import 'superdesk-core/scripts/core/itemList';
+import 'superdesk-core/scripts/core/menu';
+import 'superdesk-core/scripts/core/notify';
+import 'superdesk-core/scripts/core/ui';
+import 'superdesk-core/scripts/core/upload';
+import 'superdesk-core/scripts/core/lang';
+import 'superdesk-core/scripts/core/config';
+import 'superdesk-core/scripts/core/loading';
+
+import 'superdesk-core/scripts/apps/workspace';
+import 'superdesk-core/scripts/apps/dashboard';
+import 'superdesk-core/scripts/apps/users';
+//import 'superdesk-core/scripts/apps/groups';
+//import 'superdesk-core/scripts/apps/products';
+import 'superdesk-core/scripts/apps/publish';
+import 'superdesk-core/scripts/apps/templates';
+import 'superdesk-core/scripts/apps/profiling';
+import 'superdesk-core/scripts/apps/desks';
+import 'superdesk-core/scripts/apps/authoring';
+import 'superdesk-core/scripts/apps/search';
+import 'superdesk-core/scripts/apps/legal-archive';
+//import 'superdesk-core/scripts/apps/stream';
+import 'superdesk-core/scripts/apps/packaging';
+import 'superdesk-core/scripts/apps/highlights';
+import 'superdesk-core/scripts/apps/translations';
+import 'superdesk-core/scripts/apps/content-filters';
+import 'superdesk-core/scripts/apps/dictionaries';
+import 'superdesk-core/scripts/apps/vocabularies';
+import 'superdesk-core/scripts/apps/archive';
+import 'superdesk-core/scripts/apps/monitoring';
+import 'superdesk-core/scripts/apps/settings';
+import 'superdesk-core/scripts/apps/ingest';
+//import 'superdesk-core/scripts/apps/search-providers';
```

Comment:
This is actually quite helpful, it allows us to keep track of what superdesk modules we can safely disable without breaking the app